### DISCUSSION TITLE
Why are first letters in menu bar underlined?

### DISCUSSION THREAD

**Discussion by reagle** [saulpw/visidata#1464] [Q&A] [answered]
I would think this means that once the menubar is active (`ctrl+h`) I can press those letters to go to their sub-menus.

**Comment by anjakefala** [comment, marked as answer]
I believe that is what they mean, and I think there is a bug!

**Reply by anjakefala** [reply]
Ah! `Alt+<key>` will take you to that menu item whether inside or outside the menu. That keystroke pattern is not immediately clear. At minimum, I will add some indication to the manpage.

**Reply by reagle** [reply]
Something that confuses me is if I hit `alt+c` for the column, and then change my mind and want to go to the row menu:

1. `r` does nothing;
2. `alt+r` calls a `row key by regex` command; I suspect the menu ate the `alt` key and escaped the menu and then executed `r`.

This is confusing because is the menu system eating characters or not?

**Reply by anjakefala** [reply]
Please open a bug report!

**Reply by reagle** [reply]
Done #1470 .
